lib/libutee/include/tee_api.h

```diff
@@ -33,6 +33,9 @@
 #include <compiler.h>
 #include <tee_api_defines.h>
 #include <tee_api_types.h>
+#if defined(CFG_TEE_PANIC_DEBUG)
```

Comment:
It's an advantage when you can tell what a flags means by the name instead of having to grep through all the source code to know.

`CFG_TEE_PANIC_DEBUG` is something that is safe to have enabled even in production units (`CFG_TEE_TA_LOG_LEVEL` decides if anything is logged any way). `CFG_TEE_TA_DEBUG` on the other hand suggests that all kind of debug stuff that isn't safe for production units could be enabled.

I don't care that much how we do it I would even be OK with `CFG_TEE_TA_LOG_LEVEL >= 3` even if it's a bit to much spaghetti for my taste.